### ISSUE TITLE
Climate sensor availability

### ISSUE THREAD

**Issue by betizad** [rospogrigio/localtuya#261]
Do you have any estimate on when climate sent will be available in the library?
I found this library but the version of pytuya that it uses does not match localtuya and it does not work together with this library.

https://github.com/renat85/localtuya_thermostat

**Comment by postlund** [comment]
We have our own version of pytuya that we develop ourselves (it's derivative work from another version). Currently @rospogrigio is the only one working on that (see #72). But questions regarding "when is X ready" are generally not something anyone want to answer as that depends on a lot of (often personal) factors. It will be ready in due time, unless @ros

**Comment by rospogrigio** [comment]
Yeah I am quite overwhelmed at the moment, maybe I can find some time during the holidays but cannot guarantee... will keep you posted.

**Comment by betizad** [comment]
Thanks for working on this. I get the point on the update status and I see that you are super active on the repo. 
If it can be any help, I have two BHT-002 WiFi floor water heating device. I can gladly do testing and report feedback.

**Comment by rDuinoStar** [comment]
> Thanks for working on this. I get the point on the update status and I see that you are super active on the repo.
> If it can be any help, I have two BHT-002 WiFi floor water heating device. I can gladly do testing and report feedback.

Any feedback?

**Comment by colethegamer123** [comment]
BHT-002 would be also important for me. Thank you for your hard work. We wait as much is needed :)

**Comment by ed-holland** [comment]
I have a TH213 and have determined the following values:
![image](https://user-images.githubusercontent.com/53995991/121805840-43625d00-cc4d-11eb-815b-4a874647f8fe.png)
There are 4 not clear, but the others are available in the smart life app.

**Comment by edudele** [comment]
> BHT-002 también sería importante para mí. Gracias por tu duro trabajo. Esperamos lo que sea necesario :)

+1
